File: Segundo-Bimestre/Parcial 2/Unidad9-Exceptions/Unidad9-Exceptions_java/src/unidad9/exceptions/Ejercicio4.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package unidad9.exceptions;

import java.util.Scanner;

/**
 *
 * @author DELL
 */
public class Ejercicio4 {
    public static void main(String[] args) {
        
        try{
            Scanner leer = new Scanner(System.in);
            
            System.out.print("Ingrese el valor 1: ");
            int valor1 = leer.nextInt();
            
            System.out.print("Ingrese el valor 2: ");
            int valor2 = leer.nextInt();

            int resultado = valor1/valor2;
            
            System.out.println("El resultado es: "+resultado);            
        }
        
        /**catch(ArithmeticException ExpresionAritmetica) {
            System.out.println("Lo sentimos hay un error");
            System.out.println("de tipo "+ExpresionAritmetica);
        }*/
        
        /**catch(InputMismatchException ExcepcionErrorEntrada){
            System.out.println("Lo sentimos hay un error de entrada");
            System.out.println("de tipo "+ExcepcionErrorEntrada);
        }  */
        
        catch(Exception e){
            System.out.println("Lo sentimos hay un error de tipo "+e);
        }
    }    
}

File: Segundo-Bimestre/Parcial 2/Unidad9-Exceptions/Unidad9-Exceptions_python/Ejercicio4.py
print("")
# Inicializar las variables
valor1 = 0; valor2 = 0; resultado = 0

# Proceso
try:
    valor1 = int(input("Ingrese el valor 1: ")) 
    valor2 = int(input("Ingrese el valor 2: ")) 

    resultado = valor1/valor2

    print("El resultado es: ",resultado)

except Exception as e:
    print("")
    print("Lo sentimos, hay un error de tipo",e)
    print("")
